MaxMind.GeoIP2/WebServiceClient.cs

```diff
@@ -93,12 +93,27 @@ public class WebServiceClient : IGeoIP2WebServicesClient, IDisposable
         /// <param name="locales">List of locale codes to use in name property from most preferred to least preferred.</param>
         /// <param name="host">The host to use when accessing the service</param>
         /// <param name="timeout">Timeout in milliseconds for connection to web service. The default is 3000.</param>
+        /// <param name="httpMessageHandler"><c>HttpMessageHandler</c> to use when <c>HttpClient</c>.</param>

```

Comment:
This sentence seems unclear to me!

Reply:
I don't think it is clear to anyone. :slightly_smiling_face:

Reply:
I force pushed a fix.